cds-snc/covid-alert-portal
# Set focus to error summary on validation errors

# Summary | Résumé

Sets focus to the Error Summary when there are validation errors on a form. We do this by:
- Setting tabindex=-1 on the summary container so the element is focusable by script
- On pageload, check to see if there is a "#validation-errors" element on the screen, and set focus to it

## To test
- Navigate to a form (try the first step, email form)
- Submit without entering anything in the field so you trigger a validation error
- Notice the focus is set to the Error Summary
